libmscore/barline.cpp

```diff
@@ -1086,6 +1160,20 @@ void BarLine::layout2()
                         bbox() |= symBbox(SymId::reversedBracketBottom).translated(-w1, y2);
                         break;
                         }
+                  case BarLineType::END_START_REPEAT:
```

Comment:
I'm kind of doubting this code is needed, layout2 is for updating barlines after system layout as we don't necessarily know the height of barline until then.